h2/utilities.py

```diff
@@ -329,6 +329,13 @@ def _reject_pseudo_header_fields(headers, hdr_validation_flags):
             seen_pseudo_header_fields
         )
 
+    # If ':status' pseudo-header is not there in a response header, reject it
+    if (hdr_validation_flags.is_response_header and
+        not any(field in seen_pseudo_header_fields
+                for field in (b':status', u':status'))):
```

Comment:
My original thinking was to do something more like:

```python
if (b':status' not in seen_pseudo_header_fields) and (u':status' not in seen_pseudo_header_fields)
```

subject to line length. I suspect that inlining this way is more efficient than creating tuples or using `any()`, but that's just a gut feeling.